app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/card_components/Card.jsx

```diff
@@ -0,0 +1,137 @@
+        padding: 0,
+        minHeight: "auto",
+      };
+    }
+  };
+
+  const setStyleWithReordering = (style) => {
+    /** Moving element inside accepted column */
+    return {
+      userSelect: "none",
+      padding: 0,
+      minHeight: "auto",
+      ...style,
+    };
+  };
+
+  return (
+    <Draggable
+      key={item.id}
+      draggableId={item.id}
+      index={index}
+      isDragDisabled={item.status !== false}
+    >
+      {(provided, snapshot) => {
+        return (
+          <div
+            className={`rounded-lg h-20 space-y-3 mb-2 focus:outline-none relative justify-center ${
+              sidebarCollapsed
+                ? "d:w-24"
+                : "w-80 border border-gray-tint-2 mx-5"
+            }
+            ${item.expanded ? "h-auto" : "w-24"}`}
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+            style={
+              item.status === false
+                ? setStyleWithReordering(provided.draggableProps.style)
+                : setStyleWithoutReordering(
+                    provided.draggableProps.style,
+                    snapshot
+                  )
+            }
+          >
+            {sidebarCollapsed ? (
+              <CardImage
+                img={isAchievement ? item.badge_image_url : item.image_url}
+                sidebarCollapsed={sidebarCollapsed}
+                isUserAction={isAchievement ? false : true}
+              ></CardImage>
+            ) : isAchievement ? (
+              <AchievementCard
+                categories={categories}
+                achievement={item}
+                sidebarCollapsed={sidebarCollapsed}
+                handleUncompleteAction={handleUncompleteAction}
+              ></AchievementCard>
+            ) : (
+              <UserActionCard
+                categoryColor={categoryColor}
+                userAction={item}
+                handleDelete={handleDelete}
+                handleCompleteAction={handleCompleteAction}
+                sidebarCollapsed={sidebarCollapsed}
+              ></UserActionCard>
+            )}
+
+            {!sidebarCollapsed && (
+              <div>
+                <div
+                  className=" h-14 w-full top-0 mt-6 absolute justify-center"
+                  onClick={() => handleExpanded(item, !item.expanded)}
+                >
+                  <button
+                    className={`fas right-0 focus:outline-none mr-4 absolute ${
+                      item.status === true ? "mt-1" : "mt-4"
+                    } ${item.expanded ? "fa-chevron-up" : "fa-chevron-down"} `}
+                    onClick={() => handleExpanded(item, !item.expanded)}
+                  ></button>
+                  <div
+                    className={`font-bold text-left text-sm mt-1.5 ml-20 w-3/5 ${
+                      isAchievement && "-mt-2"
+                    }`}
+                  >
+                    {item.name}
```

Comment:
If we keep the button I think this div should be placed before the chevron so that it follows the order in the item (easier to find).